tests/crowdsourcing/tasks/turn_annotations_static/expected_states/in_flight_qa.json

```diff
@@ -0,0 +1,190 @@
+{"inputs": [
```

Comment:
This feels like redundancy from the pytest regressions I'm adding

Reply:
Hmm - but do you have a sense that the pytest regressions work that you're doing on ParlAI tasks could also be applied to Mephisto tasks, which have a much different structure? For instance, there isn't a concept of looping over examples of a dataset with a Mephisto task like there is with a ParlAI task